packages/clay-pagination/src/Item.tsx

```diff
@@ -21,20 +21,16 @@ const ClayPaginationItem = ({
 	href,
 	...otherProps
 }: IPaginationItemProps) => {
-	const ElementTag = href ? ClayLink : 'button';
-
 	return (
 		<li className={classNames('page-item', {active, disabled})}>
-			<ElementTag
-				type={href ? undefined : 'button'}
+			<ClayLink
```

Comment:
I think to make sure this doesn't break some applications in DXP, we can add an `onClick` to `ClayLink` to do `event.preventDefault()` when `href` is not set and call `onClick` from `otherProps ` if defined.

This way we guarantee that the applications that were paging on the client side do not break.

Reply:
@matuzalemsteles this is the only change I couldn't make. It was breaking too many tests and the links example would reload the url instead of updating the pagination.

Reply:
@pat270 hmm, this is weird, I guess it shouldn't break, have you tried something similar to this?

```jsx
<ClayLink
  onClick={(event) => {
    if (!href) {
      event.preventDefault();
    }

    if (otherProps.onClick) {
      otherProps.onClick(event);
    }
  }}
>
  {children}
</ClayLink>
```

Reply:
@matuzalemsteles Thanks, I didn't include the `otherProps.onClick`.